Configuration/Geometry/python/GeometryExtended2023D17Reco_cff.py

```diff
@@ -19,11 +19,11 @@
 from Geometry.CaloEventSetup.CaloTopology_cfi import *
 from Geometry.CaloEventSetup.CaloGeometryBuilder_cfi import *
 CaloGeometryBuilder = cms.ESProducer("CaloGeometryBuilder",
-    SelectedCalos = cms.vstring("HCAL"                   ,
-                                "ZDC"                    ,
-                                "EcalBarrel"             ,
-                                "TOWER"                  ,
-                                "HGCalEESensitive"       ,
+    SelectedCalos = cms.vstring("HCAL",
+                                "ZDC",
+                                "EcalBarrel",
+                                "TOWER",
+                                "HGCalEESensitive",
                                 "HGCalHESiliconSensitive" 
```

Comment:
test is failing now because there is a space after the last " here (and in the other 2023 reco geom cffs), but not in the dict file. Not sure why the modernizer didn't delete that space.